s3tests/readwrite.py

```diff
@@ -195,7 +195,7 @@ def main():
             warmup_pool = gevent.pool.Pool(size=100)
             for file_name in file_names:
                 fp = next(files)
-                warmup_pool.spawn_link_exception(
+                warmup_pool.spawn(
```

Comment:
I know `spawn_link_exception` is deprecated from 0.13, but is it supposed to be replaced by spawn? (iirc there is something like `link_exception`)